docker/images/aha/Dockerfile

```diff
@@ -1,20 +1,11 @@
 # vim:set ft=dockerfile:
-FROM vertexproject/vtx-base-image:py38
+FROM synbuild:base
```

Comment:
Doesn't this make the Docker container build dependent on a local environment condition ( the synbuild:base tag being built/correct )?

Reply:
It does break the dockerfiles from being independent, instead using shell scripts to run the builds for the containers.

This lets us easily layer the `vertexproject/synapse-aha` image ontop of the `vertexproject-synapse` image easily, since those application specific images just have entrypoints defined and shouldn't need to re-install the base synapse package each time. The time to build the entire suite of containers is now pretty much the same as it takes to build a single container.